src/StagedProposalProcessor.sol

```diff
@@ -394,10 +426,11 @@ contract StagedProposalProcessor is IProposal, PluginUUPSUpgradeable {
         Proposal storage proposal = proposals[_proposalId];
         Stage storage stage = stages[proposal.stageConfigIndex][proposal.currentStage];
 
-        // Prevent submission if `stageDuration` has passed.
-        if (proposal.lastStageTransition + stage.stageDuration < block.timestamp) {
-            revert Errors.StageDurationAlreadyPassed();
-        }
+        // // Prevent submission if `stageDuration` has passed.
+        // TODO: We remove this so optimistic plugin can still submit even after stageDuration ended.
+        // if (proposal.lastStageTransition + stage.stageDuration < block.timestamp) {
+        //     revert Errors.StageDurationAlreadyPassed();
+        // }
```

Comment:
I wold remove this, maybe adding a comment regarding results can always be reported

Reply:
Yes, Planning on removing this for sure once the PR is ready to merge.